Automattic/wp-calypso
# Site Assembler: Turn on inline-styles and color-and-fonts flags on wp-calypso

Related to #

## Proposed Changes

* Turn on the `pattern-assembler/sidebar-revamp` and `pattern-assembler/color-and-fonts` flags on wp-calypso for stakeholders to test easily
* Turn on the `pattern-assembler/inline-styles` flag on all environments excluding production for further testing

## Testing Instructions



* Use the Calypso Live link and head to `/setup?siteSlug=<your_site>`
  * Note that the site should be the site with a free plan
* Continue until you land on the Design Picker
* On the Design Picker screen, scroll down to the bottom and select BCPA CTA
* On the Pattern Assembler screen
  * Open the network panel and ensure `/sites/<your_site_id>/block-renderer/settings` endpoint returns the inline styles instead of the external links. See D101700-code for more details. 
  * Ensure you're able to see the revamp sidebar
  * Ensure you're able to see the colors & fonts

## Pre-merge Checklist
